util/pytest_log_filter.py

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on 02 02 2022
+
+@author: Kornee Kleijwegt
+
+Copyright © 2022 Kornee Kleijwegt, Delmic
+
+This file is part of Odemis.
+
+Odemis is free software: you can redistribute it and/or modify it under the terms of the GNU General Public License version 2 as published by the Free Software Foundation.
+
+Odemis is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License along with Odemis. If not, see http://www.gnu.org/licenses/.
+
+----------------------------------------------------------------------------------------------------
+
+This file allows to filter a pytest test report such that only the errors, failures etc. are outputted.
+This means that the part of the report regarding the passed tests and on the current progress are filtered out.
+
+Run the script by calling it with as first (and only) input argument the location of a text based file where the pytest test report can be found.
+    > ./pytest_log_filter.py ~/location/test_file.txt
```

Comment:
doesn't this need python3 in front of it? `python3 pytest_log_filter.py ~/location/test_file.txt`

Reply:
Because the shebang is written to execute via python this is not necessary.